Lunch/ViewController.swift

```diff
@@ -24,5 +31,37 @@ class ViewController: UIViewController {
     override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
         
     }
+
+    func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return self.items.count
+    }
+
+    func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
+        var cell:UITableViewCell = self.lunchTableView.dequeueReusableCellWithIdentifier("LunchItemCell") as! UITableViewCell
```

Comment:
It's more conventional to put a space after the color, e.g. `var cell: UITableViewCell = …`